src/ucs/stats/stats.c

```diff
@@ -327,6 +327,7 @@ static void ucs_stats_add_to_filter(ucs_stats_node_t *node,
     int found = 0;
     int filter_index = 0;
     int i;
+    unsigned num_counters_out;
```

Comment:
why do you need this variable?

Reply:
True, it's not needed anymore - Removed it.